content/_code-samples/claim-payment-channel/js/claimPayChannel.ts

```diff
@@ -0,0 +1,74 @@
+import {
+  AccountObjectsRequest,
+  Client,
+  PaymentChannelCreate,
+  PaymentChannelClaim,
+  hashes,
+} from 'xrpl'
+
+const client = new Client('wss://s.altnet.rippletest.net:51233')
+
+void claimPayChannel()
+
+// The snippet walks us through creating and claiming a Payment Channel.
+async function claimPayChannel(): Promise<void> {
+  await client.connect()
+
+  // creating wallets as prerequisite
+  const { wallet: wallet1 } = await client.fundWallet()
+  const { wallet: wallet2 } = await client.fundWallet()
+
+  console.log('Balances of wallets before Payment Channel is claimed:')
+  console.log(await client.getXrpBalance(wallet1.classicAddress))
+  console.log(await client.getXrpBalance(wallet2.classicAddress))
+
+  // create a Payment Channel and submit and wait for tx to be validated
+  const paymentChannelCreate: PaymentChannelCreate = {
+    TransactionType: 'PaymentChannelCreate',
+    Account: wallet1.classicAddress,
+    Amount: '100',
+    Destination: wallet2.classicAddress,
+    SettleDelay: 86400,
+    PublicKey: wallet1.publicKey,
+  }
+
```

Comment:
You probably should add a `console.log` before you do `submitAndWait` in all of these snippets to let people running the snippet know to expect something. (It should also specify what the sent transaction is)

Ex. `console.log("Submitting a PaymentChannelCreate transaction...")